### PR TITLE
Docs: Add info about version guarantees.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: docs/getting_started/install.md [hunk 1]
```diff
@@ -7,25 +7,48 @@ GluonTS is available from PyPi via:
...
+```{caution}
 
-This means that to use most models and features additional dependencies need to
-be installed. See the next section for more information.
+GluonTS uses [Semantic Versioning](https://semver.org) for managing versions.
+Since the library is actively developed we use `v0` as the major version.
+
+We plan to release a new minor version at the end of each quarter. The current
+minor release is `0.10`, with `0.11` being scheduled to be released end of
+September 2022.
```
Comment (lostella) [inline docs/getting_started/install.md]: I think this information should not go there: maybe we can link to the milestones on GitHub instead?

Comment (jaheba) [reply docs/getting_started/install.md]: Changed it.

File: docs/getting_started/install.md [hunk 2]
```diff
@@ -7,25 +7,47 @@ GluonTS is available from PyPi via:
 pip install gluonts
 ````
```
Comment (lostella) [inline docs/getting_started/install.md]: Four backticks closing?

File: docs/getting_started/install.md [hunk 3]
```diff
@@ -7,25 +7,47 @@ GluonTS is available from PyPi via:
...
 
-```{attention}
-**GluonTS uses a minimal dependency model.**
+```{caution}
```
Comment (lostella) [inline docs/getting_started/install.md]: Does this `{caution}` block ever close?

**Comment by jaheba** [comment]
Looks like there was an error before. Magically, it worked before. Now it should also be up to spec.

**Review by lostella — DISMISSED** [review]

**Review by lostella — APPROVED** [review]

**Pull request merged**